client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -0,0 +1,93 @@
+import { isP2Plus } from '@automattic/calypso-products';
+import { translate } from 'i18n-calypso';
+import moment from 'moment';
+import ThankYouV2 from 'calypso/components/thank-you-v2';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { preventWidows } from 'calypso/lib/formatting';
+import { useSelector } from 'calypso/state';
+import { getSiteOptions } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import ThankYouPlanProduct from '../products/plan-product';
+import type { ReceiptPurchase } from 'calypso/state/receipts/types';
+
+interface PlanOnlyThankYouProps {
+	primaryPurchase: ReceiptPurchase;
+}
+
+export default function PlanOnlyThankYou( { primaryPurchase }: PlanOnlyThankYouProps ) {
+	const isMonthsOld = ( rawDate: string, months: number ) => {
+		if ( ! rawDate || ! months ) {
+			return false;
+		}
+
+		const parsedDate = moment( rawDate );
+		return moment().diff( parsedDate, 'months' ) > months;
+	};
```

Comment:
```suggestion
const isMonthsOld = ( rawDate: string, months: number ) => {
	if ( ! rawDate || ! months ) {
		return false;
	}

	const parsedDate = moment( rawDate );
	return moment().diff( parsedDate, 'months' ) > months;
};

export default function PlanOnlyThankYou( { primaryPurchase }: PlanOnlyThankYouProps ) {
```
No need to define `isMonthsOld` within the component.

Reply:
Also take a look please at my comment below with explanation regardinng optional parameters